### DISCUSSION TITLE
Problems with directivity baloon file

### DISCUSSION THREAD

**Discussion by maxencedkk** [Ifsttar/I-Simpa#360] [Q&A]
Hello ! 

I am currently working on a studio project with lots of speakers. I have created a matlab code allowing me to write a file exactly in the same format as example 3 of isimpa giving the directivities of the speakers according to the frequency. (I am attaching the text file). 

[directivities.txt](https://github.com/Universite-Gustave-Eiffel/I-Simpa/files/11452068/directivities.txt)

I then associated this file with my sources. 

![image](https://github.com/Universite-Gustave-Eiffel/I-Simpa/assets/80481348/209654a6-22c8-4974-b766-f52ec001f5fc)

However when I launch the SPPS calculation nothing works. 

Maybe the issue comes from de properties ? 

Thank you for your potential help.

**Comment by danroth-nyt** [comment]
@maxencedkk This is cool.  What console messages are you returning from your SPPS tests?  Also, are you able to share the Matlab script you're using to generate this and where you place it to have it read in I-Simpa?  Thanks.

**Reply by sidway** [reply]
Hello there. 

I tested this feature a while ago and it does works. But I think the direction it's inverted. Try to point to the opposite direction. 

Also, before using in a real case. Test in a big room with surface receiver on both sides so you can see the sound propagation to check if its being directed.

Good luck. Best Regards,
Sidney

**Comment by maxencedkk** [comment]
> @maxencedkk This is cool. What console messages are you returning from your SPPS tests? Also, are you able to share the Matlab script you're using to generate this and where you place it to have it read in I-Simpa? Thanks.

Hey sorry for the (very) late answer ! Here is the matlab code, I didn't open it for a long time ! 

[Directivity.zip](https://github.com/Universite-Gustave-Eiffel/I-Simpa/files/14279630/Directivity.zip)
